### ISSUE TITLE
Pyflakes detects wrong line in code analysis errors compared to ipython

### ISSUE THREAD

**Issue by karlaspuldaro** [elyra-ai/elyra#1400]
**Describe the issue**
The static code analysis tool used in LSP extension detects error in the wrong code line compared to ipython
![image](https://user-images.githubusercontent.com/25207344/110502607-594e0800-80c9-11eb-9ef9-ff10cf066d8e.png)

**To Reproduce**
1. Open a notebook (or python editor in Elyra)
2. Add a few lines of python code that includes a syntax error
3. Run the notebook cell (or run your script from python editor)
4. Notice the syntax error message from ipython kernel and the code line of the error
5. Open the LSP diagnostics panel (right click on the editor)
6. Notice the same syntax error pointing to a different code line

**Environment information**
 - Elyra version: 2.1.0rc2
 - Operating system: MacOS
 - Browser: Chrome

**Comment by lresende** [comment]
Is this the same as https://github.com/elyra-ai/elyra/issues/719

**Comment by karlaspuldaro** [comment]
@lresende Not really, pyflakes seems to be always detecting the line above where the actual problem is, independent of blank lines in the file, like in this notebook example:
![image](https://user-images.githubusercontent.com/25207344/111210093-33ca6e00-85a3-11eb-8c83-38028bba1121.png)

Then when the error in on the 1st line it will point to line 0:
![image](https://user-images.githubusercontent.com/25207344/111210314-7b50fa00-85a3-11eb-8e86-035cf8d53117.png)
